src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -1,18 +1,18 @@
 /**
```

Comment:
Looks like your IDE reformatted the file. Are you able to revert this change?

Reply:
I actually formatted it on purpose, is there an issue with that?

Reply:
Overall, no, but it's a bit out of the scope of this specific PR, so it might be better to do in a separate one.

Reply:
Okay